### PR TITLE
Hello-Microverse Project Setup

### PR DESCRIPTION
**Pull request by amon-cofie**
This is my first pull request for my first repository in the Microverse Program. This is my Hello Microverse Project. I've done the following things in it

- Added a markup file (HTML)
- Added a stylesheet file (CSS)
- Added a .gitignore file
- Added test.md file in .gitignore file
- Setup HTML,CSS linters
- Added a descriptive Readme file
- Added license for this project

### CODE REVIEW

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @amon-cofie ,


Great job so far on setting up the Hello Microver Project. 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Descriptive PR. ✔️
- Nice trial with the Project setup ✔️



## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,44 @@
...
+- HTML and CSS
+- Linters
+
+## Live Demo (if available)
+
+[Live Demo Link](http://192.168.8.100:5500/Hello-Microverse/)
+
+## Getting Started
+
+**Just open it up and check it out.**
```
[the remark this answers is not in the document]
Comment (amon-cofie) [reply README.md:16]: Oh okay. Thank you @zieeco

File: README.md [hunk 2]
```diff
@@ -1,2 +1,44 @@
...
+
+## Authors
```
[the remark this answers is not in the document]
Comment (amon-cofie) [reply README.md:18]: Okay will do @zieeco

File: README.md [hunk 3]
```diff
@@ -1,2 +1,46 @@
...
+
+## Getting Started
+
+- Clone this repository to your local computer.
+- Open the "index.html" file with your web browser.
+
+**Pre-requisites.**
+
+- Knowledge of Basic HTML syntax
+- Knowledge of Basic CSS syntax
```
Comment (cynthiainga) [inline README.md:18]: - [ ] I think this part it's related to what the users need in their machine to make the project work or to be able to clone the repo, for example users can need git installed or to have VS Code in ther machine or some thing like that 😁

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @amon-cofie, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- [ ] No linters errors ✔️ 
- [ ] Used correct Github flow ✔️ 
- [ ] Descriptive PR ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1,2 +1,46 @@
...
+
+## Built With
+
+- HTML and CSS
+- Linters
+
+## Getting Started
+
+- Clone this repository to your local computer.
+- Open the "index.html" file with your web browser.
```
[the remark this answers is not in the document]
Comment (amon-cofie) [reply README.md:13]: If you could help me with some suggestions I'll be very happy, considering I have one more review😅😅😅😅. Thank you @cynthiainga

File: README.md (hunk 3 above)
Comment (amon-cofie) [reply README.md:18]: Thank you very much @cynthiainga

**Review by Munsa1 — APPROVED** [review]
Hi @amon-cofie, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**